apisix/plugins/request-id.lua

```diff
@@ -0,0 +1,67 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core          = require("apisix.core")
+local plugin_name   = "request-id"
+local ngx           = ngx
+local uuid          = require("resty.jit-uuid")
+
+local schema = {
+    type = "object",
+    properties = {
+        header_name = {type = "string", default = "X-Request-Id"},
+        include_in_response = {type = "boolean", default = false}
+    }
+}
+
+
+local _M = {
+    version = 0.1,
+    priority = 11010,
+    name = plugin_name,
+    schema = schema,
+}
+
+
+function _M.check_schema(conf)
+    return core.schema.check(schema, conf)
+end
+
+
+function _M.rewrite(conf)
+    local headers = ngx.req.get_headers()
+    uuid.seed()
```

Comment:
we have set the right `seed`, so we do not need to set it again:

https://github.com/apache/apisix/blob/master/apisix/init.lua#L71

let we remove it.